.github/workflows/underlay-test.yaml

```diff
@@ -16,15 +16,15 @@ on:
       - 'docs/**'
   workflow_dispatch:
 jobs:
-  test:
+  underlay-test:
```

Comment:
I'm not certain, but I think these job names may be involved in the GH branch protection rule on the `main` branch. If so, you may get blocked from merging until we update the rule.

Reply:
The status checks are tied to the entire workflow and not the individual jobs. Hence this change will not have any impact on PR checks.
Reason for change - this displays convenient name on the logs screen. 

also tested - in verily-src/terra-cost-billing-service

- verified that branch protection settings have status checks needed
- Create PR which changes job name (https://github.com/verily-src/terra-cost-billing-service/pull/27). Merged the PR. The required status changes are as expected. 
- Create another PR with minor readme changes (https://github.com/verily-src/terra-cost-billing-service/pull/28). Status changes are as expected.

Reply:
There may be two stale checks, which i suspect will be fixed when this is merged. 
in the case this is blocker, I can revert the changes

Reply:
see https://github.com/DataBiosphere/tanagra/pull/887 -- once the workflows are updated in main, there should be no issues

Reply:
reverted